src/sharding.js

```diff
@@ -128,52 +127,45 @@ class ShardingDatastore {
   }
 
   query (q /* : Query<Buffer> */) /* : QueryResult<Buffer> */ {
-    const tq/* : Query<Buffer> */ = {
-      keysOnly: q.keysOnly
+    let tq/* : Query<Buffer> */ = {
```

Comment:
/let/const/

Reply:
I'm a bit old-school, I use const when the object won't change; which is the JS usage..